doc/build.md

```diff
@@ -2,17 +2,23 @@
 
 ## Required Components
 You will need the following tools and components to build the Basketweave-S (not included in the kit):
+![components](https://imgur.com/C8kzkQa.png)
 - Soldering iron and solder
 - Phillips head screwdriver
 - Flush side cutters
-- No-clean flux (recommended to prevent bridging on USB pins)
-- 70 MX style switches (or 71 when not using the encoder)
-- 5 PCB mount 2u stabilizers
+- No-clean flux (heavily recommended to prevent bridging on USB pins)
+- 69-72 MX style switches (1 less per encoder, 1 more if you use split backspace)
+- 4-5 PCB mount 2u stabilizers (1 less if you use split backspace)
 - Keycaps (check compatibility, this layout uses many non-standard keys)
 - USB Type C cable
+- (Optional) Some sort of desoldering tool
+- (Optional) A switch puller is better if you want to use Millmax sockets
+
+
 ## Included Components
-The following components are included in the Basketweave keyboard kit:
-![kit](https://i.imgur.com/a4ZWz0k.jpg)
+The following components are included in the Basketweave-S keyboard kit:
+(The picture includes millmax sockets and stabilizers that can also be purchased alongside the 42 kit but are not included)
```

Comment:
Why include the Mill Max sockets in the picture and then add the clarification? I'd prefer not having them in the picture if possible.